### PR TITLE
Add field import_url to Project

### PR DESCRIPTION
**Pull request by apricote**
Field is available in the API, but missing from the api docs.

https://gitlab.com/gitlab-org/gitlab/-/blob/b93c0cf331413309e44695dac7a3bc63f5bf1b5d/lib/api/entities/project.rb#L98-100

### CODE REVIEW

**Comment by svanharmelen** [comment]
I'm a bit reluctant to support an undocumented field as we do not know if the field is actually part of the public API and might still change before it gets (if at all) added to the docs... So in the past we declined adding undocumented fields and I think I want to stick to that for now.

**Comment by svanharmelen** [comment]
I thought about it a little more, but decided that for now I want to stick to only supporting things that are documented... Sorry!

**Comment by apricote** [comment]
> I thought about it a little more, but decided that for now I want to stick to only supporting things that are documented... Sorry!

Makes sense for the project! No problem :)

**Comment by timofurrer** [comment]
@apricote I went ahead and documented the field https://gitlab.com/gitlab-org/gitlab/-/merge_requests/111933

@svanharmelen any chance to re-open and merge this once the aforementioned MR is merged?

**Comment by svanharmelen** [comment]
Sure thing 👍🏻

**Comment by timofurrer** [comment]
@svanharmelen The MR has been merged. Can we merge and release this?

**Review by svanharmelen — APPROVED** [review]

**Comment by svanharmelen** [comment]
Tagged `v0.80.2`

**Pull request merged**
